external/CMakeLists.txt

```diff
@@ -200,6 +169,9 @@ if(MSVC)
 	endif()
 endif()
 
+set(UNICORN_ARCH "arm")
+add_subdirectory(unicorn EXCLUDE_FROM_ALL)
+
```

Comment:
I don't know if it's needed, but if this add new project into solution then please add `set_property(TARGET unicorn PROPERTY FOLDER externals)` to put it into `externals` folder in MSVC

Reply:
Ah yes, good catch, i have forget it